src/app/components/music-controls/music-controls.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="controls-wrapper">
+    <div class="play-controls">
+        <button class="transparent_button" (click)="playPreviousTrack()">
+            <i class="material-icons" id="arrow">arrow_left</i>
```

Comment:
`id` is a unique identificator. You shouldn't repeat it's value. Use class instead